### PR TITLE
Ft retrive all posts

### PR DESCRIPTION
**Pull request by ishimwebernard**
#### What does this PR do?
[what does this PR do?]
>this PR allows users to get al posts from the database
#### Description of Task to be completed
[the tasks that you completed on the feature]
>added JavaScript method to trace the routes to get all posts
image

### CODE REVIEW

File: Server/controllers/postsController.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const Posts = require('../models/Posts')
+
+class Post{
+     static async savePost(request, response){
+        const toPost = new Posts({
+            title: request.body.title,
+            description: request.body.description,
+            imagesource: request.body.imagesource
+        });
+        console.log(request);
```
Comment (habinezadalvan) [inline Server/controllers/postsController.js:11]: please remove the unnecessary console.log

File: Server/controllers/postsController.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        await toPost.save();
+        response.json(toPost);  }  
+
+
+        static retrieveAllPosts =  function(request, response){
+            console.log(`Retrieving Comments ...`);
```
Comment (habinezadalvan) [inline Server/controllers/postsController.js]: remove unnecessary console.log

**Review by habinezadalvan — COMMENTED** [review]
work on the comment

File: Server/controllers/postsController.js (hunk 2 above)
Comment (ishimwebernard) [reply Server/controllers/postsController.js]: Removed unnecessary console.log

**Pull request merged**